package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "leannky-Anty",
+  "version": "1.0.0",
+  "description": "most of women rights get lost becuse they dont know their legal rights specially inheritance rights,also women cann't resort to legal consultant on the real life such as fair,or not has mony.",
+  "main": "index.js",
+  "scripts": {
+    "start": "node src/index.js",
+    "dev": "nodemon src/index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FACG6/leannky-Anty.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/FACG6/leannky-Anty/issues"
+  },
+  "homepage": "https://github.com/FACG6/leannky-Anty#readme",
+  "dependencies": {
+    "body-parser": "^1.18.3",
+    "cookie-parser": "^1.4.4",
+    "dotenv": "^7.0.0",
```

Comment:
can you explain what 'dotenv' did?

Reply:
it is used for environment variables, we need it in database connection with heroku database

Reply:
please, merge the PR if there is no any issue.
thanks

Reply:
it's new to me.
but we can deal with the DB variables without it dotenv.